docs/gpa.md

```diff
@@ -14,6 +14,28 @@ This document annotates selected parts of the GPA, with details on how to publis
 
    References from Articles VII:4, VII:5, IX:8, IX:9, X:7, X:9, X:11, XIII:2, XIV:1, XVI:2 are not yet added.
 ```
+<div markdown=1>
+
+# Common operations
+
+To avoid repetition in the guidance, we refer and link to the following common operations.
+
+## Create a release
+
+1. Set [`id`](https://standard.open-contracting.org/latest/en/schema/identifiers/#release-id) to the [release ID](https://standard.open-contracting.org/latest/en/schema/identifiers/#release-id).
```

Comment:
Keep only the second link, to avoid linking to the same page twice.

Reply:
Done